### PR TITLE
turn off firefox for travisci

### PR DESCRIPTION
**Pull request by mjneil**
https://github.com/travis-ci/travis-ci/issues/8242

### CODE REVIEW

File: test/karma.conf.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(config) {
 
   // On Travis CI, we can only run in Firefox.
```
Comment (gesinger) [inline test/karma.conf.js]: This comment is a bit out of date. Would be worth mentioning why firefox isn't included.

**Comment by forbesjo** [comment]
You can use a simpler Chrome set up now https://github.com/forbesjo/mpd-parser/blob/master/.travis.yml#L9
And maybe headless Chrome https://github.com/forbesjo/mpd-parser/blob/master/test/karma.conf.js

File: test/karma.conf.js [hunk 2]
```diff
@@ -14,8 +14,11 @@ module.exports = function(config) {
   };
 
   // On Travis CI, we can only run in Firefox.
+  // TODO: This should be using firefox. It is currently turned off because
+  //       of https://github.com/travis-ci/travis-ci/issues/8242 When this issue is
+  //       resolved, this should be updated back to latest
```
Comment (gesinger) [inline test/karma.conf.js]: This should probably just say to include 'Firefox'

File: test/karma.conf.js (hunk 1 above)
Comment (gesinger) [reply test/karma.conf.js]: Probably best to remove this comment.

File: test/karma.conf.js [hunk 3]
```diff
@@ -13,9 +13,11 @@ module.exports = function(config) {
     }
   };
 
-  // On Travis CI, we can only run in Firefox.
+  // TODO: This should include firefox. It is currently turned off because
+  //       of https://github.com/travis-ci/travis-ci/issues/8242 When this issue is
+  //       resolved, this should be updated back to latest
```
Comment (gesinger) [inline test/karma.conf.js]: minor: but in this scenario it should just say `this should be updated to include firefox` instead of the mention of latest

Comment (gesinger) [reply test/karma.conf.js]: Or just remove everything after the issue link

**Comment by forbesjo** [comment]
Bump "karma-chrome-launcher" to "^2.2.0" to get headless chrome support

**Pull request merged**